README.md

```diff
@@ -65,6 +65,11 @@ const job = await client.submitJobLocalFile("./path/to/file.mp4", {
     segments_to_transcribe: [{
         start: 1.0,
         end: 2.4
+    }],
+    speaker_names: [{
+        display_name: "steve"
```

Comment:
Because these are the same strings that will be displayed to human transcriber and returned in transcript, it should be capitalized and we should have an example
```suggestion
        display_name: "Steve"
        },{
        display_name: "Alex"
```